tvb_build/docker/Dockerfile-Windows-build

```diff
@@ -19,7 +19,7 @@ RUN activate && conda install -y --name tvb-run -c conda-forge jupyterlab flask
 
 RUN activate tvb-run && pip install --upgrade pip
 RUN activate tvb-run && pip install certifi
-RUN activate tvb-run && pip install h5py>=2.10 formencode cfflib jinja2 nibabel sqlalchemy==1.1.14 sqlalchemy-migrate==0.11.0 allensdk
+RUN activate tvb-run && pip install h5py>=2.10 formencode cfflib jinja2 nibabel sqlalchemy==1.3.20 sqlalchemy-migrate==0.13.0 allensdk
```

Comment:
Could it work without the specific sqlalchemy and migrate version Numbers (latest versions will be taken, like for the rest of packages)?